src/server/generic_family.cc

```diff
@@ -1497,9 +1500,9 @@ OpResult<uint32_t> GenericFamily::OpStick(const OpArgs& op_args, ArgSlice keys)
 
   uint32_t res = 0;
   for (uint32_t i = 0; i < keys.size(); ++i) {
-    auto [it, _] = db_slice.FindExt(op_args.db_cntx, keys[i]);
-    if (IsValid(it) && !it->first.IsSticky()) {
-      it->first.SetSticky(true);
+    auto find_res = db_slice.FindMutable(op_args.db_cntx, keys[i]);
```

Comment:
I suspect we have a bug here that we dont remove the expire info if there is

Reply:
1. You mean from the expiry dash table, right?
2. Good catch!
3. That's regardless of this PR, right?

Reply:
yes from the expire dash table. Sure this bug was not introduced by this PR

Reply:
Filed https://github.com/dragonflydb/dragonfly/issues/2315